src/current/v23.2/show-virtual-cluster.md

```diff
@@ -0,0 +1,127 @@
+---
+title: SHOW VIRTUAL CLUSTER
+summary: The SHOW VIRTUAL CLUSTER statement provides details on virtual clusters and physical replication streams.
+toc: true
+docs_area: reference.sql
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} The `SHOW VIRTUAL CLUSTER` statement lists the virtual clusters running in a CockroachDB cluster. `SHOW VIRTUAL CLUSTER` only supports inspecting virtual cluster status as part of the [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) workflow.
+
+## Required privileges
+
+`SHOW VIRTUAL CLUSTER` requires the `admin` role or the `MANAGEVIRTUALCLUSTER` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges).
+
+Use the [`GRANT SYSTEM`]({% link {{ page.version.version }}/grant.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SYSTEM MANAGEVIRTUALCLUSTER TO user;
+~~~
+
+{% include enterprise-feature.md %}
+
+## Synopsis
+
+<div>
+{% include {{ page.version.version }}/physical-replication/show-virtual-cluster-diagram.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------+------------
+`virtual_cluster_spec` | The name of the virtual cluster.
+`REPLICATION STATUS` | Display the details of a replication stream.
+`CAPABILITIES` | Display the [capabilities]({% link {{ page.version.version }}/create-virtual-cluster.md %}#capabilities) of a virtual cluster.
+
+
+{% comment %}Responses and Data state are a copy from the monitoring page, these should be in an include once the first PR has merged.{% endcomment %}
+## Responses
+
+The following table is a list of all the responses returned with the different `SHOW VIRTUAL CLUSTER` statements:
+
+Field    | Response
+---------+----------
+`id` | The ID of the virtual cluster.
+`name` | The name of the standby (destination) virtual cluster.
+`data_state` | The state of the data on the virtual cluster. This can show one of the following: `initializing replication`, `ready`, `replicating`, `replication paused`, `replication pending cutover`, `replication cutting over`, `replication error`. Refer to [Data state](#data-state) for more detail on each response.
+`service_mode` | The service mode shows whether the virtual cluster is ready to accept SQL requests. This can show one of the following: <br>`none`<br>`shared`: The virtual cluster's SQL connections will be served by the same nodes that are serving the system interface.
```

Comment:
I think this line has some formatting issues .. is the `:` a stray? I think you probably wanted another `<br>` there.

Reply:
I adapted this. It actually wasn't a formatting issue, but realized it wasn't working out how I wanted it to.